linera-core/src/chain_worker/actor.rs

```diff
@@ -153,7 +153,7 @@ where
     StorageClient: Storage + Clone + Send + Sync + 'static,
     ViewError: From<StorageClient::StoreError>,
 {
-    worker: ChainWorkerState<StorageClient>,
+    state: ChainWorkerState<StorageClient>,
```

Comment:
I'd rather not rename this. We already overuse "state" quite a bit, and we were initially tending more towards renaming the `ChainWorkerState` to `ChainWorker` instead.

Reply:
Ah, I didn't know we were about to rename `ChainWorkerState`.  I'll do that instead :)